sdcm/tester.py

```diff
@@ -888,6 +888,12 @@ def setUp(self):  # pylint: disable=too-many-branches,too-many-statements
                 validate_raft_on_nodes(nodes=db_cluster.nodes)
 
     def set_system_auth_rf(self, db_cluster=None):
+
+        def _nodetool_repair(node):
+            self.log.info('starting repair system_auth keyspace on node %s ...', node.name)
+            node.run_nodetool(sub_cmd="repair -pr system_auth")
```

Comment:
we have the nodetool events, not sure if we need those info print per node (maybe debug would be enough)

Reply:
possibly not, I'll remove it